### PR TITLE
Add section about peer-dependencies

### PR DESCRIPTION
**Pull request by ivarni**
I was thinking something like this can remedy the issue in #415. The only problem is... The depended URL does not show peerDepedencies and `ffe-form` is showing up because it has an optionalDependency so for this to work we'd need to use both.

Optional means that npm will try to install it but the process won't fail if it can't. I'm not sure what exactly happens if the project already depends on a different version of said package, that's maybe something we should test.

So in short, this can't be merged yet, we need to figure out a way to detect packages that will need updates. I just figured I'd get the ball rolling on this.

### CODE REVIEW

**Review by selbekk — APPROVED** [review]
Do we do this before or after we release the new breaking version?

**Comment by ivarni** [comment]
There's a new breaking version?

**Comment by selbekk** [comment]
> If you are making any breaking changes that will lead to a new major release, please
> visit the package's dependents and update their peerDependencies

Do we follow your instructions in the pull request that leads to the breaking change, or directly after?

**Comment by ivarni** [comment]
In the same PR so it all gets released at the same time?

**Comment by ivarni** [comment]
Shouldn't matter what order the commits are in, if that's what you meant.

**Comment by selbekk** [comment]
As long as our release process doesn't crash because we're referencing versions that aren't available at the time of release, that sounds fine with me.

**Comment by ivarni** [comment]
I'll have to defer to the people who know how the release process works to answer that.

**Review by antidecaf — APPROVED** [review]

**Comment by ivarni** [comment]
@selbekk why on earth would you merge this?

> So in short, this can't be merged yet, we need to figure out a way to detect packages that will need updates. I just figured I'd get the ball rolling on this.

This isn't adressed yet.

> As long as our release process doesn't crash because we're referencing versions that aren't available at the time of release, that sounds fine with me.

It does.

**Comment by selbekk** [comment]
Oh, fair enough. I took @antidecaf 's approval as an OK that it didn't indeed crash. It's been reverted.

**Comment by ivarni** [comment]
Well, with the PR closed at least we can leave this behind us. We still have an unsolved issue with peer dependencies though.
